GANDLF/training_loop.py

```diff
@@ -480,16 +480,16 @@ def training_loop(
         training_data_for_torch,
         batch_size=params["batch_size"],
         shuffle=True,
-        pin_memory=params["in_memory"],
+        pin_memory=params["in_memory_gpu"],
```

Comment:
https://stackoverflow.com/a/55564072/6784556

Can we take a look into this?

Reply:
Sure, that makes sense. This is the reason why I put `in_memory_gpu` as a separate option compared to `in_memory`.